packages/marketplace/tests/fixtures.ts

```diff
@@ -0,0 +1,48 @@
+import {ethers} from 'hardhat';
+import {Contract, Signer} from 'ethers';
+
+export async function deploy(
+  name: string,
+  users: Signer[] = []
+): Promise<Contract[]> {
+  const Contract = await ethers.getContractFactory(name);
+  const contract = await Contract.deploy();
+  await contract.deployed();
+  const ret = Array();
+  for (const s of users) {
+    ret.push(await contract.connect(s));
+  }
+  ret.push(contract);
+  return ret;
+}
+
+export async function deployWithProxy(
```

Comment:
@mvanmeerbeck this is Andres's fix to avoid using `hardhat-upgrades`. Pls confirm whether you want to keep it or move to `hardhat-upgrades` for testing upgradeable contracts. See example here --> https://github.com/thesandboxgame/sandbox-smart-contracts/blob/530aeb59c667a7bf17c9b6bb75827c8893ae760d/packages/asset/test/fixtures/asset/assetFixture.ts#L140